src/GovUk.Education.ExploreEducationStatistics.Notifier/README.md

```diff
@@ -0,0 +1,44 @@
+# Notifier Functions
+
+This project contains Azure Functions responsible for notifying users of changes to their subscribed publications using the Gov.UK Notify service.
```

Comment:
Something along the lines of `responsible for managing subscriptions to publications and notifying users of changes to their subscribed publications using the Gov.UK Notify service` would be more appropriate because it also contains functions that allow users to subscribe and unsubscribe to the publications.